p4-16/spec/P4-16-spec.mdk

```diff
@@ -2234,7 +2233,7 @@ underlying type.
 
 Compiler implementations are expected to raise an error if the fixed-width integer representation
 for an enumeration entry falls outside the representation range of the underlying
-type.
+type. Compiler also raises an error if a `type` is used with a serializable `enum`.
```

Comment:
If this sentence is meant to restrict what kinds of types the `typeRef` can be in the syntax, then I don't think it belongs here.  It, or something like it, should be with the rest of the description of restrictions on `typeRef` above.